src/client/containers/CandidateList/CandidateList.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { useCandidates } from '../UseHooks/useCandidates';
+import { useUpdateBallots } from '../UseHooks/useUpdateBallots';
+import { CardItemDecorator } from '../../components/CandidateCard/CandidateCardItem.component';
+import { candidateCardSorting } from '../../components/CandidateCard/CandidateCardSorting';
+import {
+  DragAndSortAdapter,
+  SortableItem,
+} from '../DragAndSortAdapter/DragAndSortAdapter';
+import { onDragEnd } from '../DragAndSortAdapter/OnDragEnd';
+import PropTypes from 'prop-types';
+
+export default function CandidateList({ userId, boardId }) {
+  const { candidates, error } = useCandidates(userId, boardId);
+  const [updatedCandidates, setUpdatedCandidates] = useState(candidates);
```

Comment:
I'm not able to use candidates because of the pending promise as I understood. May I ask for your help here?

Reply:
Explain in more in detail please